include/mega/http.h

```diff
@@ -91,10 +91,10 @@ namespace mega {
 #define APISSLEXPONENTSIZE "\x03"
 #define APISSLEXPONENT "\x01\x00\x01"
 
-#define MEGA_DNS_SERVERS "2001:678:25c:2215::554,89.44.169.136," \
-                         "2001:67c:1998:2212::13,31.216.148.13," \
-                         "2405:f900:3e6a:1::103,31.216.148.11," \
-                         "2403:9800:c020::43,122.56.56.216"
+#define MEGA_DNS_SERVERS "2001:2:0:1baa::592c:a98d,89.44.169.136," \
+                         "2001:2:0:1baa::42cb:7f0e,89.44.169.141," \
+                         "2001:2:0:1baa::42cb:7f10,66.203.127.16," \
+                         "2001:2:0:1baa::592c:a988,66.203.127.14"
```

Comment:
Actually, the second update we applies for DNS servers was wrong, since the IPs were returned by the system's DNS resolver on a NAT64 network.
We need to revert commit aaef23473f957804350f4672b984261373dea50b

Reply:
Done!